dali/daliadmin/daliadmin.cpp

```diff
@@ -148,6 +149,8 @@ int main(int argc, const char* argv[])
     StringBuffer tmps;
     for (int i=1;i<argc;i++) {
         const char *param = argv[i];
+        if (startsWith(param, "--"))
```

Comment:
NB: this is added so --config=/etc/config/.. can be added to the command line, which will be needed on a AKS system, to pickup plane configuration (from global) for path striping.

Reply:
Do we need to add param to params here ?
params.append(param);

Reply:
ok - I see now loadConfiguration() uses argv so we just skip param here.